cypress/integration/rules.spec.js

```diff
@@ -0,0 +1,27 @@
+import { shouldBeRedirectedToLogin } from '../support/assertions';
+
+describe('Rules page', () => {
+  it('when not logged in it should redirect to login page', () => {
+    cy.visit('#/rules');
+    shouldBeRedirectedToLogin();
+  });
+
+  describe('when logged in', () => {
+    beforeEach(() => {
+      cy.login();
+    });
+
+    it('should display a link to view rules', () => {
+      cy.visit('/');
+      cy.get('nav').contains('Rules').should('exist');
+    });
+
+    it('should display a list of rules', () => {
```

Comment:
I would really like this test to check that the listed rule has a link to the corresponding provider and collection. And that the rule itself is clickable and has a link to the rule page.